### ISSUE TITLE
Document usage of variable "asn" in configuration

### ISSUE THREAD

**Issue by bluikko** [alice-lg/alice-lg#71]
There are a few issues that refer using the `asn =` configuration option as a list and then somehow that list of ASNs can be used in large communities.

But I could not figure out how to even use the simple `asn = 65534` in communities.

Say, I have:
```
[server]
...
# In reality I'm testing with a 32-bit ASN
asn = 65534
```
And then a large community beginning with this ASN:
```
[bgp_communities]
65534:123:45 = Some community
```

If I understand correctly, somehow the `asn` variable could be used in place of the ASN in the community.

I have tested:
```
asn:123:45 = ...
ASN:123:45 = ...
$asn:123:45 = ...
:123:45 = ...
$ASN:123:45 = ...
```
None of these work and the large community 65534:123:45 is undetected in Alice-LG.

Am I understanding the purpose of the `asn` variable correctly? What is the syntax of using it?

This should be documented in my opinion - or an example of usage in the example config file.

Some other points, I'd prefer not to open more issues without feedback if these are valid issues or user errors:
- At de-cix Alice-LG instance the communities drop-down list has colorized communities. Is this a feature of Alice-LG or themed or a specific modification at de-cix?
- At Alice-LG startup I am seeing `Skipping malformed BGP community:` but no community is listed. This seems to refer to a community coming from BIRD (BIRD 2.0.8) and not configuration error.
- With BIRD2 the IPv4 and IPv6 features are merged. This leads to the IPv4 birdwatcher listing all IPv6 neighbors with 0 routes - and conversely the IPv6 birdwatcher instance lists all IPv4 neighbors with 0 routes.

**Comment by annikahannig** [comment]
The way this variable is used is a bit unfortunate and has some legacy background. This should be cleaned up to support multiple asns.

**Comment by annikahannig** [comment]
It is not used in the communities.
The communities provide a text mapping with a wildcard and $1, $2, ($3) matching the parts of the community representation: 1234:*:789 = "text $2 $1"
Will be used in the frontend to evalutate the community "1234:56:789" to "text 789 56"

**Comment by annikahannig** [comment]
> With BIRD2 the IPv4 and IPv6 features are merged. This leads to the IPv4 birdwatcher listing all IPv6 neighbors with 0 routes - and conversely the IPv6 birdwatcher instance lists all IPv4 neighbors with 0 routes.

This seems like a bug. Is the configration for `single_table` or `multi_table`?

**Comment by bluikko** [comment]
> It is not used in the communities.
> The communities provide a text mapping with a wildcard and $1, $2, ($3) matching the parts of the community representation: 1234:*:789 = "text $2 $1"
> Will be used in the frontend to evalutate the community "1234:56:789" to "text 789 56"

Yes I found out about `$1` etc, thanks. So I guess I have misunderstood the `asn` configuration parameter. It is not exactly clear what it is used for then... I myself do not need multiple ASNs.

> This seems like a bug. Is the configration for single_table or multi_table?

It is a single table setup.
